client/src/main/scala/drt/client/components/UpdateStaffForTimeRangeForm.scala

```diff
@@ -39,14 +39,30 @@ object IUpdateStaffForTimeRangeData {
       obj.startTimeAt.utc.toDate().getUTCSeconds.toInt
     ).getTime()
 
-    val combinedEndTime: UndefOr[Double] = new Date(
-      obj.startDayAt.year(),
-      obj.startDayAt.month(),
-      obj.startDayAt.date(),
-      obj.endTimeAt.utc.toDate().getUTCHours.toInt,
-      obj.endTimeAt.utc.toDate().getUTCMinutes.toInt,
-      obj.endTimeAt.utc.toDate().getUTCSeconds.toInt
-    ).getTime()
+    val combinedEndTime: UndefOr[Double] = {
+      val startTime = obj.startTimeAt.utc.toDate().getTime()
+      val endTime = obj.endTimeAt.utc.toDate().getTime()
+      val is24Hours = (endTime - startTime) == 24 * 60 * 60 * 1000
```

Comment:
This will fail for clock change dates when there is either 23 or 25 hours in the day